db-service/lib/common/DatabaseService.js

```diff
@@ -70,6 +70,10 @@ class DatabaseService extends cds.Service {
         get '$user.locale'() {
           return _set(this, '$user.locale', ctx.locale || cds.env.i18n.default_language)
         },
+        // REVISIT: should be decided in spec meeting for definitive name
+        get '$user.now'() {
+          return _set(this, '$user.now', (ctx.timestamp || new Date()).toISOString())
+        },
```

Comment:
Likely `$now` as it's already supported in the compiler. 
But why do we need that in the database?

Reply:
P.S. What is the costs for session context variables in HANA and PostgreSQL? → can we add them like hell, or is there a price tag?